examples/cms-agilitycms/pages/api/preview.ts

```diff
@@ -19,6 +23,6 @@ export default async function handler(req, res) {
   res.setPreviewData({})
 
   // Redirect to the slug
-  res.writeHead(307, { Location: validationResp.slug })
+  res.writeHead(307, { Location: req.query.slug })
```

Comment:
```suggestion
  if (!('slug' in validationResp)) {
    throw new Error('invariant missing slug in validation response')
  }
  res.writeHead(307, { Location: validationResp.slug })
```
I think we still want to use `validationResp.slug` here as it helps stress we should avoid redirecting to un-sanitized user input.

Reply:
Thanks! I felt slimy trusting un-sanitized user input, so good to know about `validationResp.slug`.